oracle/orchestrator/orchestrator.go

```diff
@@ -91,131 +92,31 @@ func NewProviderOrchestrator(
-			base.WithIDs[mmtypes.Ticker, *big.Int](market.GetTickers()),
-			base.WithMetrics[mmtypes.Ticker, *big.Int](o.providerMetrics),
-		)
-		if err != nil {
-			return ProviderState{}, fmt.Errorf("failed to create %s's provider: %w", cfg.Name, err)
-		}
-	case cfg.WebSocket.Enabled:
-		queryHandler, err := o.createWebSocketQueryHandler(cfg, market)
-		if err != nil {
-			return ProviderState{}, fmt.Errorf("failed to create %s's web socket query handler: %w", cfg.Name, err)
-		}
-
-		provider, err = types.NewPriceProvider(
-			base.WithName[mmtypes.Ticker, *big.Int](cfg.Name),
-			base.WithLogger[mmtypes.Ticker, *big.Int](o.logger),
-			base.WithWebSocketQueryHandler(queryHandler),
-			base.WithWebSocketConfig[mmtypes.Ticker, *big.Int](cfg.WebSocket),
-			base.WithIDs[mmtypes.Ticker, *big.Int](market.GetTickers()),
-			base.WithMetrics[mmtypes.Ticker, *big.Int](o.providerMetrics),
-		)
-		if err != nil {
-			return ProviderState{}, fmt.Errorf("failed to create %s's provider: %w", cfg.Name, err)
-		}
-	default:
-		return ProviderState{}, fmt.Errorf("provider %s has no enabled query handlers", cfg.Name)
-	}
-
-	return ProviderState{
-		Provider: provider,
-		Market:   market,
-		Enabled:  len(market.GetTickers()) > 0,
-		Cfg:      cfg,
-	}, nil
-}
-
-// GetProviderState returns all of the providers and their state.
-func (o *ProviderOrchestrator) GetProviderState() map[string]ProviderState {
+// GetProviders returns all of the providers.
+func (o *ProviderOrchestrator) GetPriceProviders() []types.PriceProviderI {
 	o.mut.Lock()
 	defer o.mut.Unlock()
 
-	// Copy the providers
-	providers := make(map[string]ProviderState, len(o.providers))
-	maps.Copy(providers, o.providers)
-
-	return providers
-}
-
-// createAPIQueryHandler creates a new API query handler for the given provider configuration.
-func (o *ProviderOrchestrator) createAPIQueryHandler(
-	cfg config.ProviderConfig,
-	market types.ProviderMarketMap,
-) (types.PriceAPIQueryHandler, error) {
-	if o.apiQueryHandlerFactory == nil {
-		return nil, fmt.Errorf("cannot create provider; api query handler factory is not set")
+	providers := make([]types.PriceProviderI, 0, len(o.providers))
+	for _, state := range o.providers {
+		providers = append(providers, state.Provider)
 	}
 
-	return o.apiQueryHandlerFactory(o.logger, cfg, o.apiMetrics, market)
+	return providers
 }
 
-// createWebSocketQueryHandler creates a new web socket query handler for the given provider configuration.
-func (o *ProviderOrchestrator) createWebSocketQueryHandler(
-	cfg config.ProviderConfig,
-	market types.ProviderMarketMap,
-) (types.PriceWebSocketQueryHandler, error) {
-	if o.webSocketQueryHandlerFactory == nil {
-		return nil, fmt.Errorf("cannot create provider; web socket query handler factory is not set")
-	}
+// GetMarketMapperState returns the market map provider and its state.
+func (o *ProviderOrchestrator) GetMarketMapper() *mmclienttypes.MarketMapProvider {
```

Comment:
This isnt really a "Market Mapper".  This should just be `GetMarketMap`

Reply:
it's returning a provider not the market map. the docs need to fix tho lol